### PR TITLE
Functional tests - Add new test 'Duplicate product V2'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'Duplicate product V2'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/campaigns/productV2/functional/07_duplicateProduct.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    it('should go to \'Catalog > Products\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductsPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.productsLink,
+      );
+
```
Comment (Progi1984) [inline tests/UI/campaigns/productV2/functional/07_duplicateProduct.ts]:
```suggestion
```

**Review by Progi1984 — DISMISSED** [review]
A small comment

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by boubkerbribri — CHANGES_REQUESTED** [review]
@nesrineabdmouleh please resolve conflicts

**Review by boubkerbribri — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thx @nesrineabdmouleh :)
